@here/harp-mapview/lib/MapView.ts

```diff
@@ -3840,13 +3879,20 @@ export class MapView extends THREE.EventDispatcher {
                     return;
                 }
                 this.m_scene.add(light);
+
                 if ((light as any).isDirectionalLight) {
                     const directionalLight = light as THREE.DirectionalLight;
                     // This is needed so that the target is updated automatically, see:
                     // https://threejs.org/docs/#api/en/lights/DirectionalLight.target
                     this.m_scene.add(directionalLight.target);
                 }
                 this.m_createdLights!.push(light);
+
+                const clonedLight: THREE.Light = light.clone();
+                this.m_overlayScene.add(clonedLight);
+                if (clonedLight instanceof THREE.DirectionalLight) {
+                    this.m_overlayScene.add(clonedLight.target.clone());
```

Comment:
this isn't needed as far as I can tell, threejs already clones the target. when calling the clone method, see: https://github.com/mrdoob/three.js/blob/92ad8c5f4e8a4aa666343ab92b3fd6260fe9e4ce/src/lights/DirectionalLight.js#L35

Reply:
okay, i removed it.